src/admin/dumb_components/sub_due_form.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+const { reduxForm } = require('redux-form')
+
+const sub_due_dates = ({fields: {start, end}, handleSubmit, error}) =>
+  <form className='date-boundaries-form' onSubmit={handleSubmit}>
+    <input type='text' placeholder='Start date' {...start}/>
+    {start.touched && start.error && <div>{start.error}</div>}
+    <input type='text' placeholder='End date' {...end}/>
+    {start.touched && start.error && <div>{start.error}</div>}
```

Comment:
So we need the `fields` list from `src/admin/dumb_components/sub_due_section.js`.

Say we have that variable as `fieldList` and the field component from `src/admin/dumb_components/field.js` as `Field`.

The code would look something like

```js
<form className='date-boundaries-form' onSubmit={handleSubmit}>
  fieldList.map(field =>
    <Field
      {...fieldList[field]}
      id={field}
      mode='edit'
      key={field}
    />
</form>
```